File: classes/CompetitorInfo.py
from dataclasses import dataclass

"""
Info pertaining to a competitor in the session.
"""


# Non time series data but does update periodically because of driver changes
@dataclass
class CompetitorInfo:
    carIdx: int
    userName: str
    userId: int
    teamId: int
    teamName: str
    carNumber: str
    carName: str
    carId: int
    iRating: int
    license: str
    licenseColor: int

File: server/src/main/kotlin/models/CompetitorInfo.kt
package models

data class CompetitorInfo(
    val carIdx: Int,
    val userName: String,
    val userId: Int,
    val teamId: Int,
    val teamName: String,
    val carNumber: String,
    val carName: String,
    val carId: Int,
    val iRating: Int,
    val license: String,
    val licenseColor: Int
)
